web_src/css/editor/combomarkdowneditor.css

```diff
@@ -33,6 +41,24 @@
   color: var(--color-primary);
 }
 
+.combo-markdown-editor md-header {
+  position: relative;
+}
+.combo-markdown-editor md-header::after {
+  font-size: 65%;
```

Comment:
Would use a pixel height here, percentage is inherited which is not desirable.

Reply:
59067e8bdf881c1f664df46421bf9c538afe91f8